src/Plugin/WebformElement/CivicrmContact.php

```diff
@@ -643,4 +652,26 @@ public static function validateRequired(&$element, FormStateInterface &$form_sta
     }
   }
 
+  /**
+   * Get an element's property value with translation.
+   *
+   * @param array $element
+   *   An element.
+   * @param string $property_name
+   *   An element's property name.
+   *
+   * @return mixed
+   *   A translated element's property value, translated default value, or NULL if
+   *   property does not exist.
+   */
+  public function getTranslatedElementProperty(array $element, $property_name) {
+    $prop = $this->getElementProperty($element, $property_name);
+    if (!empty($prop) && is_string($prop)) {
+      return t($prop);
+    }
+    else {
+      return $prop;
+    }
```

Comment:
@sluc23 Can we eliminate the usage of `else` here? and simply `return $prop;`?

Reply:
@jitendrapurohit the problem is that `t($value)` function returns an Exception if the `$value` is empty. `t()` it's intended to be used with literal strings, not variables.. so we need to be sure the variable has no empty string.

Reply:
re-thinking your question, the answer is yes, it could be eliminated the else and return `$prop`.. do you want me to rework it?

Reply:
reworked and rebased to latest master commit @KarinG @jitendrapurohit